### ISSUE TITLE
Add location to plotting interface

### ISSUE THREAD

**Issue by gabriel-happle** [architecture-building-systems/CityEnergyAnalyst#1349]
The comfort plots for individual buildings uses the input locator and region to check if there is a winter = `has_heating_season` and/or a summer = `has_cooling_season` in the region specific database to calculate different comfort conditions.

Plots will work fine if the Region is not chosen correctly, but numbers in tables don't make much sense.
A simple workaround is to change the config file Region manually.

not sure which milestone to use.
@daren-thomas is it easy to add the region to the plotting interface in ArcGIS?
basically I was expecting that when you change the region for the demand calculation it will stick with the config file for the rest of the scripts.
I didn't know that the arcgis resets the config file all the time.

**Comment by daren-thomas** [comment]
@gabriel-happle I'm interpreting this issue as adding a drop-down box for selecting the region used for the plots interface in ArcGIS (as opposed to printing it out on the plots generated). yes. this is quite easy to do. I'm working on a pull request right now.

**Comment by daren-thomas** [comment]
bundling this up with #1321, which already deals with changes to the plots interface.

**Comment by gabriel-happle** [comment]
thank you @daren-thomas
